packages/angular/src/app/pages/common-profile/common-profile.component.ts

```diff
@@ -0,0 +1,196 @@
+import {
+  Component, NgModule,
+} from '@angular/core';
+import { CommonModule } from '@angular/common';
+import notify from 'devextreme/ui/notify';
+import {
+  DxButtonModule,
+  DxSelectBoxModule,
+  DxTextBoxModule,
+  DxToolbarModule,
+  DxFormModule,
+  DxNumberBoxModule,
+  DxDateBoxModule,
+} from 'devextreme-angular';
+import { forkJoin } from 'rxjs';
+import { DxLoadPanelModule } from 'devextreme-angular/ui/load-panel';
+import { DxFileUploaderModule } from 'devextreme-angular/ui/file-uploader';
+import { DxScrollViewModule } from 'devextreme-angular/ui/scroll-view';
+import { PhonePipeModule } from 'src/app/pipes/phone.pipe';
+import {
+  FormPhotoModule,
+  FormTextboxModule,
+  ChangeProfilePasswordFormModule,
+  ProfileCardModule,
+  FormPopupModule,
+} from 'src/app/components';
+import { DataService } from 'src/app/services';
+
+@Component({
+  templateUrl: './common-profile.component.html',
+  styleUrls: ['./common-profile.component.scss'],
+  providers: [DataService],
+})
+export class CommonProfileComponent {
+  profileId = 22;
+
+  profileData: Record<string, any>;
+
+  isLoading = true;
+
+  supervisorsList = [];
+
+  isChangePasswordPopupOpened = false;
+
+  isDataChanged = false;
+
+  basicInfoItems: Record<string, any>[] = this.getBasicInfoItems();
+
+  contactItems: Record<string, any>[] = this.getContactItems();
+
+  addressItems: Record<string, any>[] = this.getAddressItems();
+
+  constructor(private service: DataService) {
+    forkJoin([
+      service.getSupervisors(),
+      service.getProfile(this.profileId)
+    ]).subscribe(([supervisorsList, profileData]) => {
```

Comment:
unsubscribe / take(1) is missing here

Reply:
it seems that it is not necessary, this subscriptions are completed